src/libs/actions/Report.ts

```diff
@@ -804,11 +804,11 @@ function clearAvatarErrors(reportID: string) {
  * @param participantAccountIDList The list of accountIDs that are included in a new chat, not including the user creating it
  */
 function openReport(
-    reportID: string,
+    reportID = '',
```

Comment:
We try to get rid of ids as `''`

Reply:
In this method we already have a check like
```
if (!reportID) {
        return;
    }
```

So we can just have 
```suggestion
    reportID: string | undefined,
```